mappings/ab0dec21aa7d939ad124ffaf95faf8ed72b1018ff72fffd8f9974907474f4b4559.json

```diff
@@ -0,0 +1,64 @@
+{
+    "subject": "ab0dec21aa7d939ad124ffaf95faf8ed72b1018ff72fffd8f9974907474f4b4559",
```

Comment:
Hi @craigimmel,

the website shows a different policyId: c7dcfa416c127f630b263c7e0fe0564430cfa9c56bba43e1a37c6915

Can you please change that on the website or we will close the submission in 2 days.

Thanks!

Reply:
Hey @VladislavKudrin 

Thanks for the heads up. I have updated the policy id in the website footer at https://gokey.finance. Please let me know if any additional changes are needed.

Best,
Craig
![Screenshot 2024-04-06 131928](https://github.com/cardano-foundation/cardano-token-registry/assets/90641093/b6a53b05-fb76-4d10-8fd2-7cffba85dd80)